src/pages/Home/Home.page.jsx

```diff
@@ -19,12 +20,24 @@ function HomePage({ filter }) {
     history.push('/');
   }
 
+  function redirectToLogin() {
+    history.push('/login');
+  }
   return (
-    <Section ref={sectionRef}>
-      <Container>
-        <MainDisplay state={state} filter={filter} />
-      </Container>
-    </Section>
+    <>
+      {authenticated ? (
+        <Layout>
+          <Section ref={sectionRef}>
+            <h2>Home</h2>
+            <Container>
+              <MainDisplay state={state} filter={filter} />
+            </Container>
+          </Section>
+        </Layout>
+      ) : (
+        redirectToLogin()
+      )}
+    </>
```

Comment:
Better if you evaluate `authenticated` before returning value so that the jump to login page executes before and the render looks cleaner